src/Header/Header.styles.js

```diff
@@ -1,10 +1,11 @@
 import { StyleSheet } from 'react-native';
+import { HEADER_HEIGHT } from '../utils/dimensions';
 
 const styles = StyleSheet.create({
   container: {
     flex: 1,
     flexDirection: 'row',
-    height: 50,
+    height: HEADER_HEIGHT,
```

Comment:
Do you think the header's height can be changed by dimension too?

Reply:
Not for now, I think is ok to be fixed at 50